subsys/pcd/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+
+zephyr_library()
+
+zephyr_library_sources(
+  src/pcd.c
+  )
+if(CONFIG_MCUBOOT AND CONFIG_SOC_NRF5340_CPUAPP)
```

Comment:
These lines needs to be in a later commit

Reply:
yes, need to do some micro refactoring of files to get the PR correct